kubernetes/charts/edge-kubernetes/templates/iotedged-deployment.yaml

```diff
@@ -29,6 +29,12 @@ spec:
           env:
           - name: "IOTEDGE_LOG"
             value: "debug"
+          - name: "IOTEDGE_HAS_CLUSTER_RBAC"
```

Comment:
We don't require this env var in the daemon, if we just fail gracefully always.

Reply:
I think I found a nice, clean way to do this.  It might be a little more code, but then I won't have to pass the env var.